d3d12/bors.toml

```diff
@@ -0,0 +1,5 @@
+status = [
```

Comment:
This file can definitely be nuked

Reply:
This sounds like an adjustment to make after merge, yes? As you [stated earlier](https://github.com/gfx-rs/wgpu/pull/3987#issuecomment-1736213961), our intent here is to bring in the original source as-is, unless it's breaking something, with refinements in follow-up PRs.

Reply:
Ah alright, If I said that, do that :)